_projects/2024/100535255/100535255.Rmd

```diff
@@ -0,0 +1,1059 @@
+                                   y = avg_age, yend = avg_age, 
+                                   color = category), linewidth = 0.5) + 
+  
+  geom_point(aes(awardYear, age, color = category), size = 1.5, alpha = 0.8) +
+  geom_line(aes(awardYear, age, color = category), lwd = .2) + 
+  
+  #new shape for women (diamond shape = 23)
+  geom_point(data = filter(tidy_nobels, gender == "female"), 
+             aes(x = awardYear, y = age), color = "#B55F94", 
+             shape = 23, size = 3, stroke = 0.8) +
+  facet_grid(category ~ ., switch = "y") +
+  scale_color_manual(values = colors) +
+  
+  
+  geom_text(data = min.years_label, aes(x = x, y = y, label = label), color = "#7F8082", 
+            size = 3, family = "gl", vjust = -1) +
+  geom_text(data = maj.years_label, aes(x = x, y = y, label = label), color = "#707070", 
+            size = 3, fontface = "bold", family = "gl", vjust = -1) +
+  
+  geom_text(data = labels_avg_facets, #avg_cat (59)
+            aes(x = 1898, y = label_y, label = label_text),
+            hjust = 1, vjust = 1, size = 2.5, color = "black", alpha= 0.8, 
+            fontface = "bold",family = "gl") +
+  
+  geom_text(data = labels_avg_cat_facets,
+            aes(x = 1898, y = label_y, label = label_text,fontface = "bold", family = "gl"),
+            hjust = 1, vjust = -0.8, size = 2.5, color = colors) +
+  
+  geom_text(data = label_cat,
+            aes(x = 1898, y = label_y, label = label_text,fontface = "bold", family = "tinos"),
+            hjust = 1, vjust = -0.8, size = 3, color = colors, lineheight = 0.8) +
+  
+  
+  theme_void() + 
+  theme(
+    legend.position = "none", 
+    panel.grid.major.x = element_blank(), 
+    axis.text.y = element_blank(), 
+    axis.text.x = element_blank(),
+    strip.text.y = element_blank(),
+    plot.background = element_rect(fill = "#f2ebde", color = NA),
+    panel.spacing = unit(0, "lines"),
+    plot.margin = margin(10,5,0,0, "mm")) + 
+  labs(y = "", x = "") +
+  
+  coord_cartesian(xlim = c(1900, 2012), ylim = c(23, 92), clip = "off", expand = 0)
+
+gp1
+```
+
+### Chronicle of data - final plot
+
+Included all the improvements, the final graph will be as follow:
+
+```{r final plot improved,layout="l-screen", fig.height=12, fig.width=20}
+
+colored_spacer <- plot_spacer() + 
+  theme(plot.background = element_rect(fill = "#f2ebde", color = NA))
+
+row1 <- (colored_spacer + gp1 + gp2 + gp4 + colored_spacer) + 
+  plot_layout(widths = c(4.5, 15, 1, 4.5, 0.5)) +
+  theme(plot.margin = margin(15, 0, 30, 0))
+
+row2 <- (colored_spacer + gp3 + colored_spacer)  + 
+  plot_layout(widths = c(1.35, 4.41, 1.85)) +
+  theme(plot.margin = margin(0, 0, 0, 0))
+
+row3 <- (graph_title + legend) +
+  plot_layout(widths = c(3, 4)) +
+  theme(plot.margin = margin(50, 0, 30, 0))
+
+final_plot <- (colored_spacer / row1 / row2 /row3 ) + 
+  plot_layout(heights = c(0.1, 1.3, 0.55, 0.5)) &
+  theme(
+    plot.background = element_rect(fill = "#f2ebde", color = NA))
+
+final_plot
```

Comment:
My only comment is that fonts in general are a little small in this last graph. The legend is hard to read even at full page.